src/app/access-requests/services/access-request.service.ts

```diff
@@ -205,6 +207,10 @@ export class AccessRequestService {
       if (filter.status) {
         requests = requests.filter((ar) => ar.status === filter.status);
       }
+      const requestText = filter.requestText?.trim().toLowerCase();
+      if (requestText) {
+        requests = requests.filter((ar) => ar.request_text === requestText);
```

Comment:
This should work similar to the other notes (includes instead of equal).

Reply:
You're right. And so should the ticket ID!